odoo/addons/base/models/ir_rule.py

```diff
@@ -200,7 +200,7 @@ def _make_access_error(self, operation, records):
 
         if not self.user_has_groups('base.group_no_one') or not self.env.user.has_group('base.group_user'):
             records.invalidate_recordset()
-            return AccessError(f"{operation_error}\n\n{resolution_info}")
```

Comment:
Is it intentional ? It isn't legit at all :thinking:

Reply:
Sorry, no, that was just for debugging, and the PR is not fully done.
I didn't expect robodoo to ask review from so many people :laughing: